pkg/common/consumer/consumergroup_offsets_checker.go

```diff
@@ -63,10 +64,7 @@ func (k *KafkaConsumerGroupOffsetsChecker) WaitForOffsetsInitialization(ctx cont
 			return false, fmt.Errorf("error checking if offsets are initialized. stopping trying. %w", err)
 		}
 	}
-	pollCtx, pollCtxCancel := context.WithTimeout(ctx, OffsetCheckRetryTimeout)
-	err = wait.PollUntil(OffsetCheckRetryInterval, check, pollCtx.Done())
-	defer pollCtxCancel()
-
+	err = wait.PollImmediate(OffsetCheckRetryInterval, OffsetCheckRetryTimeout, check)
```

Comment:
NIT: `check` could we name the conditional function to something more meaningful ?

like `checkIfAllOffsetsInitialized` ? 

(at least on the PR to `main` branch)

Reply:
+1. can do in main

Reply:
Addressed in main: https://github.com/knative-sandbox/eventing-kafka/pull/911